src/Controls/src/Core/Platform/GestureManager/GestureManager.Windows.cs

```diff
@@ -511,30 +481,67 @@ void OnPointerReleased(object sender, PointerRoutedEventArgs e)
 			PanComplete(true);
 		}
 
-		void OnTap(object sender, TappedRoutedEventArgs e)
+		void OnTap(object sender, RoutedEventArgs e)
 		{
 			var view = Element as View;
 			if (view == null)
 				return;
+						
+			var tapPosition = e.GetPositionRelativeToPlatformElement(Control);
+
+			if (tapPosition == null)
+				return;
 
-			var tapPosition = e.GetPosition(Control);
-			var children = (view as IGestureController)?.GetChildElements(new Point(tapPosition.X, tapPosition.Y));
+			var children =
+				(view as IGestureController)?.GetChildElements(new Point(tapPosition.Value.X, tapPosition.Value.Y))?.
+				GetChildGesturesFor<TapGestureRecognizer>(ValidateGesture);
 
-			if (children != null)
-				foreach (var recognizer in children.GetChildGesturesFor<TapGestureRecognizer>(g => g.NumberOfTapsRequired == 1))
+			if (ProgessGestureRecognizers(children))
+				return;
+
+			IEnumerable<TapGestureRecognizer> tapGestures = view.GestureRecognizers.GetGesturesFor<TapGestureRecognizer>(ValidateGesture);
+			ProgessGestureRecognizers(tapGestures);
+
+			bool ProgessGestureRecognizers(IEnumerable<TapGestureRecognizer>? tapGestures)
```

Comment:
Should this be `ProcessGestureRecognizers`?

Reply:
👍